src/main/java/com/google/javascript/gents/TypeAnnotationPass.java

```diff
@@ -162,6 +162,19 @@ public void visit(NodeTraversal t, Node n, Node parent) {
             maybeSetInlineTypeExpression(parent, n, bestJSDocInfo, true);
           }
           break;
+          // If a DEVAULE_VALUE is in a PARAM_LIST we type annotate its first child which is the
+          // actual parameter.
+        case DEFAULT_VALUE:
```

Comment:
default values can occur in ES6 destructuring too:

let {x = 0} = {};

Are sure this makes sense for those cases?